client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -570,13 +570,14 @@ function LineItemSublabelAndPrice( {
 	if ( type === 'plan' && product.months_per_bill_period && product.months_per_bill_period > 1 ) {
 		return (
 			<>
-				{ translate( '%(sublabel)s: %(monthlyPrice)s /month × %(monthsPerBillPeriod)s', {
+				{ translate( '%(sublabel)s: %(monthlyPrice)s/month × %(monthsPerBillPeriod)s', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.